packages/web/src/pages/audio-rewards-page/components/WalletManagementTile.tsx

```diff
@@ -100,24 +101,39 @@ const WalletActions = () => {
   }, [dispatch])
 
   return (
-    <div className={styles.moreOptionsSection}>
-      <Flex gap='m' w='100%' wrap={isMobileWeb() ? 'wrap' : undefined}>
+    <Flex gap='m' wrap='wrap' justifyContent='center'>
+      <Box flex={1}>
         <OptionButton
+          minWidth={150}
           disabled={!hasBalance}
           onClick={onClickSend}
           iconLeft={IconSend}
         >
           {messages.sendLabel}
         </OptionButton>
-        <OptionButton onClick={onClickReceive} iconLeft={IconReceive}>
+      </Box>
+      <Box flex={1}>
+        <OptionButton
+          minWidth={150}
+          onClick={onClickReceive}
+          iconLeft={IconReceive}
+        >
           {messages.receiveLabel}
         </OptionButton>
-        <OptionButton onClick={onClickTransactions} iconLeft={IconTransaction}>
+      </Box>
+      <Box flex={1}>
+        <OptionButton
+          minWidth={150}
```

Comment:
nit: magic number

Reply:
prefer const?

Reply:
i'd say so but keep it local to this file?

Reply:
Moved it out to the option button component, since it's only used when we set it to 150. ty! good callout!!